pages/theme-test.js

```diff
@@ -0,0 +1,237 @@
+	background-color: var(--surface1);
+	color: var(--text2);
+`;
+
+const Surface2 = styled(Surface)`
+	background-color: var(--surface2);
+	color: var(--text2);
+`;
+
+const Surface3 = styled(Surface)`
+	background-color: var(--surface3);
+	color: var(--text1);
+`;
+
+const Surface4 = styled(Surface)`
+	background-color: var(--surface4);
+	color: var(--text1);
+`;
+
+const TextBrand = styled.h1`
+	color: var(--brand);
+`;
+
+const Text1H1 = styled.h1`
+	color: var(--text1);
+`;
+
+const Text2H1 = styled.h1`
+	color: var(--text2);
+`;
+
+const TextOnBrandH1 = styled.h1`
+	color: var(--text-on-brand);
+	background-color: var(--brand);
+`;
+
+const BreakpointTest = styled.div`
+  width: 50px;
+  height 50px;
+  background-color: white;
+  @media (min-width: ${(props) => props.theme.breakpoints.sm}) {
+    background-color: yellow;
+    // font-size: 60%;
+  }
+  @media (min-width: ${(props) => props.theme.breakpoints.md}) {
+    background-color: blue;
+    // font-size: 55%;
+  }
+  @media (min-width: ${(props) => props.theme.breakpoints.lg}) {
+    background-color: red;
+    // font-size: 50%;
+  }
+
+`;
+
+const surfaceVariants = {
+	hidden: {
+		opacity: 0,
+	},
+	visible: {
+		opacity: 1,
+		transition: {
+			when: 'beforeChildren',
+			staggerChildren: 0.3,
+		},
+	},
+};
+
+const childrenSurfaceVariants = {
+	hidden: {
+		opacity: 0,
+		scale: 0,
+	},
+	visible: {
+		opacity: 1,
+		scale: 1,
+		transition: {
+			duration: 0.2,
+			ease: 'easeInOut',
+		},
+	},
+};
+
+export default function ThemeTest() {
+	return (
+		<StyledMain>
+			<StyledSection>
+				<SurfaceSamples
+					initial='hidden'
+					animate='visible'
+					variants={surfaceVariants}
+				>
+					<Surface1 variants={childrenSurfaceVariants}>1</Surface1>
+					<Surface2 variants={childrenSurfaceVariants}>2</Surface2>
+					<Surface3 variants={childrenSurfaceVariants}>3</Surface3>
+					<Surface4 variants={childrenSurfaceVariants}>4</Surface4>
+				</SurfaceSamples>
+			</StyledSection>
+			<StyledSection>
+				<TextSamples>
+					<TextBrand>
+						<BrandSwatch />
+						Brand
+					</TextBrand>
+					<Text1H1>
+						<Text1Swatch />
+						Text Color 1
+					</Text1H1>
+					<Text2H1>
+						<Text2Swatch />
+						Text Color 2
+					</Text2H1>
+					<TextOnBrandH1>
+						<TextOnBrandSwatch />
+						Text On Brand
+					</TextOnBrandH1>
+				</TextSamples>
+			</StyledSection>
+			<section>
+				<BreakpointTest>Text</BreakpointTest>
+				<LinkPrimarySm href='/'>
+					<a>Small</a>
+				</LinkPrimarySm>
+				<LinkPrimary href='/'>
+					<a>Primary</a>
+				</LinkPrimary>
+				<LinkPrimaryLg href='/'>
+					<a>Large</a>
+				</LinkPrimaryLg>
+				<LinkPrimaryOutlineSm href='/'>
+					<a>Small</a>
+				</LinkPrimaryOutlineSm>
+				<LinkPrimaryOutline href='/'>
+					<a>Large</a>
+				</LinkPrimaryOutline>
+				<LinkPrimaryOutlineLg href='/'>
+					<a>Large</a>
+				</LinkPrimaryOutlineLg>
+				<MotionLogo />
+			</section>
+		</StyledMain>
+	);
+}
```

Comment:
This would be an exposed route

Reply:
Didn't mean to include this either, also removed.